app/views/profiles/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="card">
+  <div class="card-body">
+    <h2 class="card-title"><%= @user.full_name %></h2>
+    <a class="card-subtitle mb-2 text-muted" href="mailto:<%= @user.email %>"><%= @user.email %></a>
```

Comment:
Apparently there is a [`mail_to` helper](https://apidock.com/rails/ActionView/Helpers/UrlHelper/mail_to) that would generate this code for us; this is IMO preferable because otherwise for example special characters in emails might cause trouble.

Reply:
Today I learned about `mail_to`! Nice, implemented in https://github.com/hpi-swt2/bookkeeper-portal-red/pull/44/commits/30f3e8fb4cc7dea36d12bb49d696821e5efb476b

Reply:
even better! done in https://github.com/hpi-swt2/bookkeeper-portal-red/pull/44/commits/5887f503fbe0fe0bbf71ad11ee033016d5b2e14c